### PR TITLE
Feat/update grants data

### PR DESCRIPTION
**Pull request by Douglasacost**
- Add db configuration file
- Update Grant contract address
- Update Grant schema (VestingSchedule)
- Modified events to only catch necessary ones
- PK now is lowercase
- Reduce workers on Docker file

### CODE REVIEW

File: subquery/schema.graphql [hunk 1]
```diff
@@ -65,9 +65,9 @@ type ERC721Token @entity @compositeIndexes(fields: [["owner", "contract"]]) {
   transfers: [ERC721Transfer!]! @derivedFrom(field: "token")
   application: String
   duration: Int
-  captureDate: BigInt @index
-  longitude: Float @index
-  latitude: Float @index
+  captureDate: BigInt
+  longitude: Float
+  latitude: Float
```
Comment (aliXsed) [inline subquery/schema.graphql:70]: I guess that's the sort of the changes that we need to reduce the unnecessary load from our indexers. Good job

File: subquery/src/mappings/click-content-sign.ts [hunk 2]
```diff
@@ -123,12 +124,10 @@ export async function handleSafeMint(tx: SafeMintTransaction) {
 
   token.uri = uri;
 
-  if (false) {
+  if (nodleContracts.includes(contract.id)) {
```
Comment (aliXsed) [inline subquery/src/mappings/click-content-sign.ts:127]: 👍

**Review by aliXsed — APPROVED** [review]
LGTM!

**Comment by Douglasacost** [comment]
Can't merge the PR, seems there are restrictions
@aliXsed Could you merge this one?

**Comment by Douglasacost** [comment]
@aliXsed could you merge this PR? I have restrictions to do so

**Pull request merged**